README.md

```diff
@@ -48,6 +50,7 @@ The [tutorial about using VON Network](docs/UsingVONNetwork.md) has information
 
 ## Running the web server in Docker against another ledger
 
+Note that in order to use Docker Desktop (> version 3.4.0), make sure you uncheck the "Use Docker Compose V2" in Docker Desktop > Preferences > General.  Refer to this issue for additional details; [#170](https://github.com/bcgov/von-network/issues/170#issuecomment-972898014)
```

Comment:
@nomadic-me, did you mean to leave the note down here as well.  I'm not sure it needs to be in both places.

Reply:
I had initially added it in both places; 
but based on your comment - it makes sense to removed the duplicate from the section "Running the web server in Docker against another ledger"